internal/tui/setup/steps.go

```
@@ -938,13 +943,26 @@ func (c confirmStep) View(width int) string {
 	}
 
-	// Triplex.
-	b.WriteString(labelStyle.Render("Triplex:"))
-	b.WriteString("\n")
-	if c.cfg.Triplex.Endpoint != "" {
+	// Extractor — show Triplex or NuExtract based on which is populated.
+	switch {
+	case c.cfg.NuExtract.Endpoint != "":
+		b.WriteString(labelStyle.Render("Extractor:"))
+		b.WriteString("\n")
+		b.WriteString(fmt.Sprintf("  Format:   NuExtract-2.0\n"))
```

golangci-lint (2.11.4)

[error] S1039: unnecessary use of fmt.Sprintf

(staticcheck)